ee/codegen_integration/conftest.py

```diff
@@ -31,6 +31,8 @@ def _get_fixture_paths(root: str) -> Tuple[str, str]:
     # https://app.shortcut.com/vellum/story/4649/remove-fixture-exclusions-for-serialization
     exclude_fixtures={
         "simple_code_execution_node",
+        # TODO: Remove after all other map node fixes go in
```

Comment:
Will remove in the parent PR / parent branch this PR is merging into. This is temporary to have CI happy